packages/shared/src/lib/lifecycle.ts

```diff
@@ -2,7 +2,8 @@
 
 declare global {
   interface Window {
-    safari?: { pushNotification?: unknown };
+    /* eslint-disable  @typescript-eslint/no-explicit-any */
+    safari?: { pushNotification: any };
```

Comment:
Had to resort to `any` with `eslint-disable`. Using `unknown` or `never` is not getting accepted by the build.

Reply:
Where is this used? Or does it break build due to internal usage by onesignal?

Reply:
`global.safari` is now added in onesignal SDK from what I can see so you can remove it from our code.

Reply:
Seems like this is a legacy code, but I removed it since it doesn't seem to be necessary anymore.